livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/realtime/realtime_model.py

```diff
@@ -767,6 +798,35 @@ def session_update(
             }
         )
 
+    def _sync_chat_ctx_to_session(
```

Comment:
Do you think this can be an async function that waits for the OAI answers?
We're currently emitting `conversation_item_deleted` and `conversation_item_created` instead.

Reply:
I guess `conversation.item.create` needs to be async?

Reply:
sounds good!

Reply:
Add a async version of item create.